README.rst

```diff
@@ -147,3 +147,52 @@ You can enable encryption for your DB by adding the following::
 
 **NOTE:** AWS does not support RDS encryption for the *db.t2.** instance classes. More details on supported instance classes are available `here <http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html>`_
 
+
+SSL cipher list pindown (updated 27/06/2015)
+============================================
+Amazon provides default policies for cipher lists:
+
+* Type: SSLNegotiationPolicyType
+* Name: Reference-Security-Policy
+
+More info:
+
+https://aws.amazon.com/blogs/aws/elastic-load-balancing-perfect-forward-secrecy-and-other-security-enhancements/
+
+http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-security-policy-options.html
+
+http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-ssl-security-policy.html
+
+http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-security-policy-table.html
+
+The policy in use currently by default is: ELBSecurityPolicy-2015-05
+This will be pinned down implicitely or explicitely with the following entry in CFN YAML, 
+part of elb -> listeners::
+
+
+		PolicyNames:
+		 - PinDownSSLNegotiationPolicy201505
+
+Example::
+
+		elb:
+		  - name: elbname
+		    hosted_zone: something.dsd.io.
+		    scheme: internet-facing
+		    certificate_name: certificate-name
+		    listeners:
+		      - LoadBalancerPort: 80
+		        InstancePort: 80
+		        Protocol: HTTP
+		      - LoadBalancerPort: 443
+		        InstancePort: 80
+		        Protocol: HTTPS
+		        # following can be commented
+		        PolicyNames:
+		         - PinDownSSLNegotiationPolicy201505
```

Comment:
It's not clear that this example is showing, or what purpose it serves. Can you explain your thinking here?